src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -498,6 +498,22 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val verboseAuditTarget: PipelineTable = PipelineTable(
+      name = "verboseAuditTarget_gold",
+      _keys = Array("organization_id", "timestamp", "unixTimeMS"),
```

Comment:
why are timestamp AND unixTimeMS BOTH keys? unixTimeMS is the same as timestamp but with slightly higher resolution -- as such remove timestamp from keys.